### PR TITLE
[FIX JENKINS-40094] Fix global config.xml content

### PR DESCRIPTION
**Pull request by amuniz**
[JENKINS-40094](https://issues.jenkins-ci.org/browse/JENKINS-40094)

@reviewbybees

### CODE REVIEW

**Comment by varmenise** [comment]
🐝

File: src/main/java/com/cloudbees/jenkins/support/api/TemporaryFileContent.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import hudson.Util;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Temporal file content, auto-deleted after {@link #writeTo(OutputStream)}.
```
Comment (batmat) [inline src/main/java/com/cloudbees/jenkins/support/api/TemporaryFileContent.java]: temporary :)

**Comment by aheritier** [comment]
🐝

**Comment by amuniz** [comment]
@reviewbybees done

**Comment by christ66** [comment]
🐝

**Pull request merged**